src/msw/anybutton.cpp

```diff
@@ -87,6 +88,25 @@ using namespace wxMSWImpl;
 extern wxWindowMSW *wxWindowBeingErased; // From src/msw/window.cpp
 #endif // wxUSE_UXTHEME
 
+// Create a disabled bitmap from the normal one taking the application mode
+// (light or dark) into account.
+static wxBitmap CreateDisabledBitmap(const wxBitmap& bmp)
```

Comment:
Just curious: Why handle it like this, instead of making `wx(Image|Bitmap)::ConvertToDisabled()` and `wxColor::MakeDisabled()` work also in (MSW) dark mode, even if dark mode had to be user set as a parameter?

Disclaimer: I have no idea how disabling works.

Reply:
> It seems wrong to have such simple functions behave differently depending on some global state

I did not meant for them to depend on a state, I meant them having an optional parameter (defaulting to light).